packages/reaction-accounts/tests/jasmine/server/integration/publications.js

```diff
@@ -1,51 +1,80 @@
-// const user = Factory.create("user");
+/* eslint dot-notation: 0 */
+
+const shopId = faker.reaction.shops.getShop()._id;
+const shopMembers = Meteor.server.publish_handlers["ShopMembers"];
+
 describe("Account Publications", function () {
-  let shop;
   beforeEach(function () {
     // reset
-    ReactionCore.Collections.Products.remove({});
-    ReactionCore.Collections.Shops.remove({});
-    ReactionCore.Collections.Orders.remove({});
-    shop = Factory.create("shop");
+    Meteor.users.remove({});
+    // ReactionCore.Collections.Products.remove({});
+    // ReactionCore.Collections.Shops.remove({});
+    // ReactionCore.Collections.Orders.remove({});
+    // shop = Factory.create("shop");
```

Comment:
If we aren't going to do this operation, can we just remove these rather than commenting them out?